src/roman_datamodels/testing/utils.py

```diff
@@ -397,7 +399,7 @@ def mk_basic_meta():
     return meta
 
 
-def mk_common_meta():
+def mk_common_meta(photometry=True):
```

Comment:
Should this be photmetry=False as the default if we are removing it as a required element? and set it to true once we want it?

Reply:
removed from `common` now